build/testing/sauce-labs/test-browsers.js

```diff
@@ -0,0 +1,199 @@
+/**
+ * @name 
+ * Test Browsers
+ * 
+ * @description 
+ * Cross browser testing with Selenium Webdriver (browser configurations) and Appium (device configurations) for testing on Sauce Labs.
+ * 
+ * @example 
+ * To execute on CLI, run 'node .circleci/run-browser-beta-tests.ts' from root directory.
+ * 
+ * @requires
+ * You must create environment variables if executing this script locally using CLI. This can be done 
+ * by adding to your '~/.bashrc' file.
+ * 
+ * export SAUCE_LABS_USER=[some value]
+ * export SAUCE_LABS_USER=[some value] 
+ *
+ * CircleCI has these values included as environment variables
+ * 
+ * @see
+ * To understand in detail:
+ * Getting Started: https://help.crossbrowsertesting.com/selenium-testing/getting-started/javascript/
+ * Scripting docs: https://www.seleniumhq.org/docs/03_webdriver.jsp#chapter03-reference 
+ */
+
+const { Builder } = require("selenium-webdriver");
+const chalk = require('chalk');
+const { Configure, Phases } = require("../config-browsers.js");
+const { spawn } = require('child_process');
+const git = spawn('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
+
+// Retrive user/key from environment variables
+const username = process.env.SAUCE_LABS_USER;
+const accessKey = process.env.SAUCE_LABS_KEY;
+const domain = "https://msft-docs.fast-dna.net";
+
+// Configure to connect to remote hub
+const remoteHub = "http://" + username + ":" + accessKey + "@ondemand.saucelabs.com:80/wd/hub";
+
+/**
+ * Get configuration based on the accepted Phase argument
+ * @param {Phases} phases
+ * @returns {Configuration based on phases}
+ */
+const getConfiguration = (phases) => {
+
+    // Parse command arguments and return valid configs
+    try {
+        switch (phases) {
+            
+            case "alpha":

```

Comment:
I agree with your suggestion, however, sentence casing takes more code since there isn't an actual function like there is for UpperCase and LowerCase so I opted for the less code solution.

Reply:
1. We could require the commands that user's pass in to just use sentence casing.  `node test-browsers.js Alpha' but this just didn't feel right.
2. We could instead pass in a char only,  `node test-browsers.js -a` but this didn't feel as intuitive and probably would then require more work to build in `--help` from the CLI on this file. 
3. Maybe there are some other work arounds?  Open to suggestions.

Reply:
Does all lower case work? I think that would be my expectation.

Reply:
converted Enum type from Any to String and eliminated lowercase as it's not needed now.